dist/index.html

```diff
@@ -157,88 +169,91 @@ <h2 class="card-title">Multi-Post <br> Stories</h3>
                          </div>
                     </div>
                   </div> 
-            </section>
+          </section>
 
-            <section  class="my-skills">
-                <div class="about"> 
-                    <div class="about-self">
-                        <div><h2 class="heading">About  Myself</h2></div>
-                        <div><p class="info">Hello I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p></div>
-                        <h3 href="#" class="connect">LET'S CONNECT</h3>
-                   <ul class="social-media">
-                      <li>
-                        <a href="#" class="icons">
-                          <img src="./img/icons/twitter.svg"/>
-                        </a>
+          <section  class="my-skills">
+            <div class="about about-skills desktop-about-skills"> 
+                <div class="about-self">
+                  <div><h2 class="heading">About  Myself</h2></div>
+                  <div>
+                    <p class="info desk-info">Hello I'm a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+                  </div>
+                  <h3 href="#" class="connect">LET'S CONNECT</h3>
+             <ul class="social-media">
+                <li>
+                  <a href="#" class="icons">
+                    <img src="./img/icons/twitter.svg"/>
+                  </a>
+                </li>
+                <li>
+                  <a href="#" class="icons">
+                    <img src="./img/icons/Linkedin.png"/>
+                  </a>
+                </li>
+                <li>
+                 <a href="#" class="icons">
+                  <img src="./img/icons/micro.svg"/>
+                </a>
+                </li>
+               <li>
+              <a href="#" class="icons"><img src="./img/icons/git.svg"/></a>
+               </li>
+               <li>
+                 <a href="#" class="icons"><img src="./img/icons/hand.svg" /></a>
+               </li>
```

Comment:
- [ ] As suggested by the previous reviewer, There's still more than required space between `About Myself` and the `Language` that need to be reduced. Kindly work on that too.

### Your Design
![your-about-spacing](https://user-images.githubusercontent.com/26597432/236509565-5b304f79-8d87-44a4-afa3-2832c137701a.png)

### Figma Design
![temp-about-spacing](https://user-images.githubusercontent.com/26597432/236509588-b74de2b0-5ee7-4540-b03c-fd9e03807e1e.png)